src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataQueueManager.java

```diff
@@ -1,7 +1,6 @@
 package com.hms_networks.americas.sc.extensions.historicaldata;
 
 import com.hms_networks.americas.sc.extensions.fileutils.FileAccessManager;
-
 import com.hms_networks.americas.sc.extensions.json.JSONException;
```

Comment:
looks like a whitespace change slipped into this commit

Reply:
yeah, that was from formatting... will move into the formatting commit